hoomd/md/external/wall.py

```diff
@@ -0,0 +1,653 @@
+# Copyright (c) 2009-2021 The Regents of the University of Michigan
+# This file is part of the HOOMD-blue project, released under the BSD 3-Clause
+# License.
+
+r"""Wall potentials.
+
+Wall potentials add forces to any particles within a certain distance,
+:math:`r_{\mathrm{cut}}`, of each wall. In the extrapolated mode, all particles
+deemed outside of the wall boundary are included as well.
```

Comment:
```suggestion
outside of the wall boundary are included as well.
```
We don't judge or consider whether particles are outside the wall. They either are or the are not.